servicemanager/service/smjvmservice.py

```diff
@@ -53,7 +52,7 @@ class SmJvmService(SmService):
 
     def __init__(self, context, service_name, expected_service_type):
         SmService.__init__(self, context, service_name, expected_service_type)
-        self.pattern = "service.manager.serviceName=%s" % self.service_name
+        self.pattern = "service.manager.serviceName=%s$" % self.service_name
```

Comment:
See  https://github.com/hmrc/service-manager/blob/8ccfd2134bf254f9eeda01ef9ec38c12011c6382/servicemanager/service/smjvmservice.py#L36 - these end up as system properties: `-Dservice.manager.serviceName=SERVICE_NAME`
So a ^ anchor would cause running services to be ignored.